lib/presentation/engage/bible_series/helper.dart

```diff
@@ -5,7 +5,11 @@ import 'package:wpa_app/domain/completions/entities.dart';
 bool isOnTime(Timestamp date) {
   DateTime now = DateTime.now();
```

Comment:
Can we convert `now` and `seriesDate` to UTC before doing the logic checks? Not sure if timezone is being taken into account here.

Reply:
ok sure